src/ui/icons/IconArrowDown.re

```diff
@@ -0,0 +1,6 @@
+[@react.component]
+let make = () => {
+  <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" className="bi bi-arrow-down" viewBox="0 0 16 16">
+    <path fillRule="evenodd" d="M8 1a.5.5 0 0 1 .5.5v11.793l3.146-3.147a.5.5 0 0 1 .708.708l-4 4a.5.5 0 0 1-.708 0l-4-4a.5.5 0 0 1 .708-.708L7.5 13.293V1.5A.5.5 0 0 1 8 1z"/>
+  </svg>
+}
```

Comment:
Is there some library where one could import the Icons from instead of defining an svg component?

Reply:
This icon is from [Bootstrap Icons](https://icons.getbootstrap.com/icons/arrow-down/). The idea was to not import the whole JS library and use only a couple of icons that were necessary, but the library can be found [here](https://getbootstrap.com/docs/5.3/getting-started/download/).